src/app/home/home.page.html

```diff
@@ -1,8 +1,65 @@
 <app-application-header [config]="headerConfig" (actionClick)="handleActionClick($event)"></app-application-header>
 <ion-content [fullscreen]="true">
-  <!-- <app-carousel [items]="carouselItems" [template]="itemTemplate" [itemsToShow]="4"></app-carousel>
+  <div class="ion-margin-vertical">
+    <app-section-header [headerName]="'My Improvements'" [showViewmore]="myImprovements.length > 0" [navigateUrl]="'/my-improvements-listing'"></app-section-header>
+    @if (myImprovements.length > 0) {
+
+      <app-carousel
+        [items]="myImprovements"
+        [template]="itemTemplate1"
+        [itemsToShow]="4"
+      ></app-carousel>
+
+      <ng-template #itemTemplate1 let-item="$item" let-index="$index">
+        <app-myimprovement-card
+        [myImprovement]="item"
+          [index]="index"
+        ></app-myimprovement-card>
+      </ng-template>
+    }
+    @else {
+      <div style="display: flex; justify-content: center;">
+        <mat-card >
+          <img class="card-image" src="assets/MI-2.0 card images/recommendedimg.png" alt="Card image" />
+          <h1 class="card-title">{{ "Create your 1st Micro Improvement" }}</h1>
+          <button class="card-button" >Create Improvement</button>
+        </mat-card>
+      </div>
+    }
+  </div>
+
+  <div class="ion-margin-vertical">
+
+
+  <app-section-header [headerName]="'Spotlights'" [showViewmore]="true" [navigateUrl]="'/spotlight-listing'"></app-section-header>
+
+  <app-carousel
+    [items]="spotlightstories"
+    [template]="itemTemplate2"
+    [itemsToShow]="4"
+  ></app-carousel>
+
+  <ng-template #itemTemplate2 let-item="$item" let-index="$index">
+    <app-spotlight-card
+      [spotlightStory]="item"
+      [index]="index"
+    ></app-spotlight-card>
+  </ng-template>
+</div>
+
+<div class="ion-margin-vertical">
+
+  <app-section-header [headerName]="'Recommendations'" [showViewmore]="true" [navigateUrl]="'/list/recommendation'"></app-section-header>
+
+  <app-carousel
+    [items]="recommendationList"
+    [template]="itemTemplate"
+    [itemsToShow]="4"
+  ></app-carousel>
+
+  <ng-template #itemTemplate let-item="$item" let-index="$index">
```

Comment:
should this ng-template be inside the carousel tags or outside, can you verify? if it can be placed inside, Please do so makes the code more intentional and readable

Reply:
i have moved the ng-template inside the app-carousel tag